Jenkinsfile

```diff
@@ -148,11 +149,15 @@ stage('Prepare') {
   }
 }
 
-stage("Sanity Check") {
+stage('Sanity Check') {
   timeout(time: max_time, unit: 'MINUTES') {
     node('CPU') {
-      ws(per_exec_ws("tvm/sanity")) {
+      ws(per_exec_ws('tvm/sanity')) {
         init_git()
+        docs = sh (returnStatus: true, script: '''
+        ./tests/scripts/git_check_tree.sh
```

Comment:
1. The name `git_check_tree` is confusing, because it only checks if the change tree includes `docs`. Better to name it `git_change_docs.sh` or something like that.
2. The result status `docs` is also confusing as it doesn't look like a flag. Maybe `only_change_docs` might be clearer.

Reply:
Fixed

Reply:
It is no longer `git_check_tree.sh`?

Reply:
This is fixed now.